### ISSUE TITLE
Feature request: bundle types into the UI kit preset 

### ISSUE THREAD

**Issue by jocelyn-dunkley** [commercetools/ui-kit#2965]
**Describe the bug**
If I want to use a type from a component I need to import the specific component package separately and reference it like this:
```
import SelectField, {
  type TCustomEvent,
} from '@commercetools-uikit/select-field';

```
But ideally I would like to put it in the same bigger import here:
```
import {
  Text,
  Spacings,
  SecondaryButton,
  MultilineTextField,
  TextField,
  NumberField,
  Grid,
} from '@commercetools-frontend/ui-kit';
```
Doing something like this doesn't seem to work:
```
import {
  Text,
  Spacings,
  SecondaryButton,
  MultilineTextField,
  TextField,
  NumberField,
  Grid,
  SelectField, {
    type TCustomEvent,
  }
} from '@commercetools-frontend/ui-kit';
```

**Expected behavior**
I want to be able to import types directly from the ui-kit package and not import the specific component each time. This disrupts the code pattern that we're trying to stick with.

**Comment by ByronDWall** [comment]
Hey @jocelyn-dunkley and @jaikumar-tj you're right that types need to be exported from the preset.

The build process for the preset makes doing exporting comprehensive types for all components in the preset a little more complex than it might seem at first.  That being said, it is now a ticket in our backlog and we will work on adding it soon.

**Comment by ByronDWall** [comment]
@jocelyn-dunkley @jaikumar-tj types should be available in UI Kit's next release (19.17.0), the [PR](https://github.com/commercetools/ui-kit/pull/3003) has been merged

**Comment by ByronDWall** [comment]
closing the issue, but please let me know if this is still an issue past the 19.17.1 release
